### PR TITLE
FIX file descriptors sizing documentation

### PR DESCRIPTION
**Pull request by fgalan**
Doc corresponding to issue #2166 

In addition, some fixes to user/README.md (missing links to two files added in a past PR).

**Note it is safe to merge this PR given it doesn't include code modifications.**

@crbrox

### CODE REVIEW

File: doc/manuals/admin/perf_tuning.md [hunk 1]
```diff
@@ -144,14 +144,41 @@ On the other hand, setting thread pools is a way of "capping" throughput. If the
...
+
+```
+max fds > 5 * n + max cons + db pool size + extra
+```
+
+where
+
+* **max fds** is the per process file descriptors limit, i.e. the output of the `ulimit -n` command. It can
+  be changed with `ulimit -n <new limit>`.
+* **n**, number of threads in the notification threads pool size. The 5 factor is due to each thread can
```
Comment (kzangeli) [inline doc/manuals/admin/perf_tuning.md]: number of threads in the notification thread pool

  or

the size of the notification thread pool

File: doc/manuals/admin/perf_tuning.md [hunk 2]
```diff
@@ -144,14 +144,41 @@ On the other hand, setting thread pools is a way of "capping" throughput. If the
...
+  hold up to 5 connections (libcurl pool).
+* **max cons** is the size of the thread pool for incoming connections, configured with `-reqPoolSize`
+  [CLI parameter](cli.md). Note that if you don't use this parameter, default is not using incoming
+  connection pool size. Thus, a burst of incoming connections large enough could exhaust in theory all
```
Comment (kzangeli) [inline doc/manuals/admin/perf_tuning.md]: default is not using any pool for incoming connections

**Comment by kzangeli** [comment]
LGTM

**Comment by crbrox** [comment]
LGTM

**Pull request merged**